gooddata-sdk/gooddata_sdk/catalog/organization/entity_model/llm_endpoint.py

```diff
@@ -0,0 +1,239 @@
+# (C) 2024 GoodData Corporation
+from __future__ import annotations
+
+from typing import Any, Optional
+
+import attr
+from gooddata_api_client.model.json_api_llm_endpoint_in import JsonApiLlmEndpointIn
+from gooddata_api_client.model.json_api_llm_endpoint_in_attributes import JsonApiLlmEndpointInAttributes
+from gooddata_api_client.model.json_api_llm_endpoint_in_document import JsonApiLlmEndpointInDocument
+from gooddata_api_client.model.json_api_llm_endpoint_patch import JsonApiLlmEndpointPatch
+from gooddata_api_client.model.json_api_llm_endpoint_patch_attributes import JsonApiLlmEndpointPatchAttributes
+from gooddata_api_client.model.json_api_llm_endpoint_patch_document import JsonApiLlmEndpointPatchDocument
+
+from gooddata_sdk.catalog.base import Base
+from gooddata_sdk.utils import safeget
+
+
+@attr.s(auto_attribs=True, kw_only=True)
```

Comment:
Consider using `define` instead of `attr.s`. `Define` is a modern way. `attr.s` is used historically.